source/assets/css/basic.less

```diff
@@ -202,6 +202,21 @@ b, strong {
     overflow-x: hidden;
 }
 
+nav.navi--main:after {
+    content: "";
+    position: absolute;
+    left: 0;
+    right: 0;
+    bottom: 50px;
+    height: 100px;
+    z-index: 10;
+    background: -moz-linear-gradient(top, rgba(20, 36, 50, 0) 0%, #142432 100%);
+    background: -webkit-linear-gradient(top, rgba(20, 36, 50, 0) 0%, #142432 100%);
+    background: linear-gradient(to bottom, rgba(20, 36, 50, 0) 0%, #142432 100%);
+    filter: progid:DXImageTransform.Microsoft.gradient(startColorstr='#00142432', endColorstr='#142432', GradientType=0);
```

Comment:
You don't need the `filter` anymore cause it targets IE6-9. The IE9 got published 6 years ago

Reply:
check